ktor-utils/ktor-utils-js/src/io/ktor/util/date/DateJs.kt

```diff
@@ -0,0 +1,34 @@
+package io.ktor.util.date
+
+import kotlin.js.*
+
+actual fun GMTDate(timestamp: Long?): GMTDate {
+    val date = timestamp?.let { Date(it) } ?: Date()
```

Comment:
How does it work? `Long` is not JS native type so how can we pass it to JS native `Date`?

Reply:
I guess it misses a `toDouble`. But I wonder about the signature. Is it a Number or dynamic? Maybe it works converting to string then to double?

Reply:
Let's add explicit `toDouble` call here